src/Microsoft.OpenApi/Models/AuthenticationScheme.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using Microsoft.OpenApi.Attributes;
+
+namespace Microsoft.OpenApi.Models
+{
+    /// <summary>
+    /// The type of HTTP authentication schemes
+    /// </summary>
+    public enum AuthenticationScheme
+    {
+        /// <summary>
+        /// Use basic HTTP authentication schemes
+        /// </summary>
+        [Display("basic")] Basic,
+
+        /// <summary>
+        /// Use bearer Authentication scheme
+        /// </summary>
+        [Display("bearer")] Bearer,
+
+        /// <summary>
+        /// Use OpenIdConnectUrl
+        /// </summary>
+        [Display("openIdConnectUrl")] OpenIdConnectUrl
```

Comment:
This is an odd one.  This isn't an HTTP auth scheme.    This is a property on the OpenAPI Security Scheme object.